fixtures/projects_templates.json

```diff
@@ -61,6 +64,9 @@
       "slug": "default",
       "template": {
         "apps": {
+          "Project Network Policy": {
```

Comment:
What is the naming convention here? The other "apps" are called `project-vol` and `project-minio`. Perhaps this should be `project-netpolicy` or something?

Reply:
Goo point. I have updated this!

Reply:
Ok nice! Looks good